component/client/settings/ClientSetting.tsx

```diff
@@ -144,7 +144,7 @@ class ClientSetting extends React.Component<any, IClientSettingState> {
 
   private reset() {
     storage.reset().then(() => {
-      storage.set(STORAGE_KEY_INITIALIZED, false);
+      storage.set(STORAGE_KEY_INITIALIZED, "false");
```

Comment:
Great, I will change StorageService to get and set any object.
@zhoutwo, so why did you make them get and set in the first place then?